### ISSUE TITLE
Persian language support

### ISSUE THREAD

**Issue by Asncodes-80** [onthegomap/planetiler#366]
Hi to PlaneTiler team!
I got Iran, Iraq tiles with PlaneTiler docker. Like `README` guideline. But name of locations like streets, cities and other doesn't show over the vector map. Also I got Monaco and Canada map without any problem, all cities name and streets name was okey.
Also I changed font of map that located in `cd "(npm root -g)/tileserver-gl-light/node_modules/tileserver-gl-styles/fonts/"` by this [repository](https://github.com/korywka/fonts.pbf), but doesn't had effect.

This is list of supported language in planetiler. But doesn't support Arabic and Persian Language. I need this scripts!
![supported_lang](https://user-images.githubusercontent.com/40909549/195974470-396288de-72f4-47ea-bf51-c505b5e2b542.png)

PlaneTiler is the best, but are there any hacky way to fix that in my use case?

**Comment by msbarry** [comment]
Hello! Thanks for reaching out.  Any of those debug logs that say `argument: <name>=...` can be overridden from the command line using `--name=value` or an env var by using `PLANETILER_NAME=value`, so if for example you wanted arabic, persion, and english you could run planetiler with `--languages=fa,ar,en`.

If you think those should be added to the default set of languages planetiler produces, I suggest making the change in [openmaptiles](https://github.com/openmaptiles/openmaptiles/blob/master/openmaptiles.yaml#L29) first, then [planetiler-openmaptiles](https://github.com/openmaptiles/planetiler-openmaptiles) will get updated from that automatically and it will change the defaults.  It looks like the defaults already contain `ar` but are missing `fa` ?

**Comment by acalcutt** [comment]
@Asncodes-80 

In tileserver, what do you see if you click the inspect button 
![image](https://user-images.githubusercontent.com/3792408/196005622-4476c5f9-ecc3-4a27-abfb-8d2519d1c14e.png)

then hover over the mouse over the road, like
![image](https://user-images.githubusercontent.com/3792408/196005643-9790445c-a570-4f1b-889f-c3d191ca6c6c.png)

**Comment by acalcutt** [comment]
No, inspect isn't what you should usually use. I was just asking in the languages you added show when you view it that way.

If they did show under inspect, you may be able to adjust your style for streets and cities to use those languages

**Comment by msbarry** [comment]
So is the issue that you do see place names and other features updated correctly, but not road names? That might be due to a limitation of openmaptiles that it doesn't includes all the name translations on transportation_name layer. I'm not quite sure why, might be to save on storage size?

We could try getting openmaptiles to fix so it shows up on maptiler and other data providers, or if not then add an extra option to planetiler to emit all road name translations, or maybe just include certain extra ones.

**Comment by acalcutt** [comment]
On my server I change my style so that it always shows the english names for places, like https://github.com/acalcutt/wifidb-tileserver-gl/blob/master/tileserver-gl/styles/WDB_OSM/style.json#L5128

On the roads, I set the text field to be {name:latin} if it exists, or {name:nonlatin} if it does not, like 
https://github.com/acalcutt/wifidb-tileserver-gl/blob/master/tileserver-gl/styles/WDB_OSM/style.json#L4608

In my screenshot of inspect above, you can see the road has name:latin and name:nonlatin, so my with my style it should show "Tehran Second Bypass", since in my style name:latin takes priority.

**Comment by acalcutt** [comment]
I actually looked at my current style and I changed that text field to this now

`"text-field": ["case",["has","name_en"],["get","name_en"],["case",["has","name:en"],["get","name:en"],["case",["has","name:latin"],["get","name:latin"],["get","name:nonlatin"]]]],`

**Comment by acalcutt** [comment]
For the Noto Sans fonts, I am using the fonts from noto-sans.zip from https://github.com/openmaptiles/fonts/releases . Have you set up a config file and set up your fonts directory?
